gcp-observability/src/test/java/io/grpc/gcp/observability/ObservabilityConfigImplTest.java

```diff
@@ -43,88 +45,159 @@
 
 @RunWith(JUnit4.class)
 public class ObservabilityConfigImplTest {
-  private static final String EVENT_TYPES = "{\n"
-      + "    \"enable_cloud_logging\": false,\n"
-      + "    \"event_types\": "
-      + "[\"GRPC_CALL_REQUEST_HEADER\", \"GRPC_CALL_HALF_CLOSE\", \"GRPC_CALL_TRAILER\"]\n"
+  private static final String LOG_FILTERS = "{\n"
+      + "    \"project_id\": \"grpc-testing\",\n"
+      + "    \"cloud_logging\": {\n"
+      + "    \"client_rpc_events\": [{\n"
+      + "        \"methods\": [\"*\"],\n"
+      + "        \"max_metadata_bytes\": 4096\n"
+      + "    }"
+      + "    ],\n"
+      + "    \"server_rpc_events\": [{\n"
+      + "        \"methods\": [\"*\"],\n"
+      + "        \"max_metadata_bytes\": 32,\n"
+      + "        \"max_message_bytes\": 64\n"
+      + "    }"
+      + "    ]\n"
+      + "    }\n"
       + "}";
 
-  private static final String LOG_FILTERS = "{\n"
-      + "    \"enable_cloud_logging\": true,\n"
-      + "    \"destination_project_id\": \"grpc-testing\",\n"
-      + "    \"flush_message_count\": 1000,\n"
-      + "    \"log_filters\": [{\n"
-      + "        \"pattern\": \"*/*\",\n"
-      + "        \"header_bytes\": 4096,\n"
-      + "        \"message_bytes\": 2048\n"
+  private static final String CLIENT_LOG_FILTERS = "{\n"
+      + "    \"project_id\": \"grpc-testing\",\n"
+      + "    \"cloud_logging\": {\n"
+      + "    \"client_rpc_events\": [{\n"
+      + "        \"methods\": [\"*\"],\n"
```

Comment:
I was also thinking of the case "*" followed by a method name e.g.

```
"methods": ["*", "service1/method4"]
```

Looking at the code, it is allowed although looks redundant. The spec also does not specifically exclude it. Should you have a test to test it (or add code to detect and flag it)?